src/WrongReturnDate.csv

```diff
@@ -0,0 +1,5 @@
+s,banana,100,2020-10-17
+b,banana,50,2020-10-15
+s,banana,100,2020-10-27
+b,banana,70,2020-10-16
+r,banana,10,2020-10-13
```

Comment:
Bogdan asked not to have this stop sign in the ned of files/program - add an empty line (in all testing files)